lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -846,6 +846,29 @@ def test_financial_assistance_form_course_exclusion(
 
         self.assertNotIn(str(course.id), response.content)
 
+    @patch.object(CourseOverview, 'load_from_module_store', return_value=None)
+    def test_financial_aid_course_overview(self, _mock_course_overview):
+        """
+        Verify that learners can not get financial aid for the courses with no
+        course overview.
+        """
+        # Create course
+        course = CourseFactory.create().id
+
+        # Create Course Modes
+        CourseModeFactory.create(mode_slug=CourseMode.AUDIT, course_id=course)
+        CourseModeFactory.create(mode_slug=CourseMode.VERIFIED, course_id=course)
+
+        # Enroll user in the course
+        CourseEnrollmentFactory(course_id=course, user=self.user, mode=CourseMode.AUDIT)
+        CourseOverview.get_from_id(course)
```

Comment:
What does this call do? Shouldn't you assert the return result to be `None`?
Where in this function to ensure `CourseOverview` object is not created?